app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayFragment.kt

```diff
@@ -257,6 +264,37 @@ class NavigationOverlayFragment : Fragment() {
         activity!!.moveTaskToBack(true)
     }
 
+    private fun observeAccountState(): Disposable {
+        val fxaRepo = serviceLocator.fxaRepo
+
+        return fxaRepo.accountState.subscribe { accountState ->
+            when (accountState) {
+                is AccountState.AuthenticatedWithProfile -> {
+                    val fxaProfile = accountState.profile
+                    if (fxaProfile.avatar != null) {
+                        PicassoWrapper.client
+                            .load(fxaProfile.avatar.url)
+                            .into(fxaButton, object : Callback {
+                                override fun onError(e: Exception?) {
+                                    Sentry.capture(e)
```

Comment:
As per my comment above about explicitly handling all states, if we fail to set the image, we should set an image here too, right?

I notice the `ImageSetStrategy` doesn't provide an option for an image to insert on error: maybe you could expand it to do so? If not, please file a follow-up bug for that!